src/bindings/python/src/openvino/frontend/tensorflow/node_decoder.py

```diff
@@ -51,13 +51,34 @@ def __init__(self, operation: tf.Operation, inner_graph: bool):
                                                     "Expected tf.Operation, got {}".format(type(operation))
         self.m_operation = operation
         self.m_inner_graph = inner_graph
+
+        # Copies value from inner buffer of TF_Operation to NodeDef class.
+        self.m_node_def = self.m_operation.node_def
```

Comment:
why do we need this?

Reply:
When we access `self.m_operation.node_def` TF copies value from inner buffer of `TF_Operation` to `NodeDef` class.
In decoder there are several places when we need to get attributes from `node_def`. So avoid redundant copy of values I save `node_def` in node decoder.